src/energyplus/Test/CentralHeatPumpSystem_GTest.cpp

```diff
@@ -184,20 +184,25 @@ TEST_F(EnergyPlusFixture, ForwardTranslatorCentralHeatPumpSystem_Nodes) {
   IdfObject i_central_hp = w.getObjectsByType(IddObjectType::CentralHeatPumpSystem)[0];
 
   // supply = Cooling
-  ASSERT_EQ(i_central_hp.getString(CentralHeatPumpSystemFields::CoolingLoopInletNodeName).get(), central_hp.supplyInletModelObject().get().name());
+  ASSERT_EQ(i_central_hp.getString(CentralHeatPumpSystemFields::CoolingLoopInletNodeName).get(),
+            central_hp.supplyInletModelObject().get().nameString());
```

Comment:
**Lots** of these, when EXPECT_EQ is used with one of two arguments being a boost::optional<T> and not a T

```
 1596 |   BOOST_STATIC_ASSERT_MSG(sizeof(CharType) == 0, "If you want to output boost::optional, include header <boost/optional/optional_io.hpp>");
```